app/[locale]/layout.tsx

```diff
@@ -87,16 +87,16 @@ export default async function RootLayout({
   const { t, resources } = await initTranslations(locale, i18nNamespaces)
 
   return (
-    <html lang="en" suppressHydrationWarning>
-      <body className={inter.className}>
+    <html lang="en" suppressHydrationWarning className="h-full">
+      <body className={inter.className + " h-full"}>
         <Providers attribute="class" defaultTheme="dark">
           <TranslationsProvider
             namespaces={i18nNamespaces}
             locale={locale}
             resources={resources}
           >
             <Toaster richColors position="top-center" duration={3000} />
-            <div className="bg-background text-foreground flex h-screen flex-col items-center">
+            <div className="bg-background text-foreground flex h-full flex-col items-center sm:h-screen">
```

Comment:
h-screen messes up with mobile. Read more here: https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser. Alternative - change everything to h-full.

Another alternative considered: h-dvh